### PR TITLE
PRIME-1723 small admin changes

### PR DESCRIPTION
**Pull request by hogmann**

### CODE REVIEW

**Review by mtpultz — DISMISSED** [review]

**Review by johnnyl6 — CHANGES_REQUESTED** [review]
Tried up/down arrow on the Notes page in PR environment, note remains unchanging after switching enrollee.

image

**Comment by hogmann** [comment]
> Tried up/down arrow on the Notes page in PR environment, note remains unchanging after switching enrollee.
> 
Alright fixed that with the note pages, as well with the upload pages.

**Review by johnnyl6 — DISMISSED** [review]
LGTM 🚀

File: prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-notes/adjudicator-notes.component.ts [hunk 1]
```diff
@@ -78,9 +78,11 @@ export class AdjudicatorNotesComponent implements OnInit {
     switch (this.noteType) {
       case NoteType.EnrolleeAdjudicationNote:
         this.getAdjudicatorNotes(this.route.snapshot.params.id);
+        this.route.params.subscribe(params => this.getAdjudicatorNotes(params.id));
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-notes/adjudicator-notes.component.ts:81]: Should quickly cast these `+`

File: prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-notes/adjudicator-notes.component.ts [hunk 2]
```diff
@@ -78,9 +78,11 @@ export class AdjudicatorNotesComponent implements OnInit {
...
         break;
       case NoteType.SiteRegistrationNote:
         this.getSiteRegistrationNotes(this.route.snapshot.params.sid);
+        this.route.params.subscribe(params => this.getSiteRegistrationNotes(params.id));
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-notes/adjudicator-notes.component.ts:85]: Should quickly cast these `+`

File: prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-adjudicator-documents/enrollee-adjudicator-documents.component.ts [hunk 3]
```diff
@@ -18,47 +17,46 @@ import { EnrolleeAdjudicationDocument } from '@registration/shared/models/adjudi
...
     private route: ActivatedRoute,
-    private utilsService: UtilsService,
-    private toastService: ToastService
-  ) {
-    this.enrolleeId = this.route.snapshot.params.id;
-  }
+    private utilsService: UtilsService
+  ) { }
 
   public onSaveDocuments(documentGuids: string[]) {
```
[the remark this answers is not in the document]
Comment (hogmann) [reply prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-adjudicator-documents/enrollee-adjudicator-documents.component.ts]: I will add a post it note to try and get engrained in my head, as well as the +

**Review by AnaisHebert — APPROVED** [review]
coffee_parrot

**Review by johnnyl6 — APPROVED** [review]

**Pull request merged**